common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -5006,6 +5011,11 @@
           "Multiple bucket TRANSFORMs."
         ]
       },
+      "MULTIPLE_QUERY_RESULT_CLAUSES_WITH_PIPE_OPERATORS" : {
```

Comment:
shall we turn it into a top-level error class? This is syntax error, not unsupported feature, and should have a different SQL state.

Reply:
Sounds good, done.